__tests__/app.test.js

```diff
@@ -549,4 +549,115 @@ describe('GET /api/users', () =>
             }    
         });
     });
+});
+
+describe('GET /api/reviews (queries)', () =>
+{
+    test('server responds with review object for a given category', () =>
+    {
+        return request(app)
+        .get('/api/reviews?category=social deduction')
+        .expect(200)
+        .then((response) =>
+        {
+            const { reviews } = response.body;
```

Comment:
We should be checking that all the reviews have the correct category if we are querying the database for this.

Reply:
updated to check for all reviews returned, thanks.